.github/workflows/wheels.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         # https://github.com/github/feedback/discussions/7835#discussioncomment-1769026
         buildplat:
           - [ubuntu-20.04, manylinux_x86_64]
-          - [macos-10.15, macosx_*]
+          - [macos-11, macosx_*]
```

Comment:
This still needs to be done in #22525. Should I add it there?

Reply:
Heh, too late. I'll just make a PR for that.